internal/operators/lso/utils.go

```diff
@@ -0,0 +1,324 @@
+package lso
+
+func Manifests() map[string]string {
+	manifests := make(map[string]string)
+	manifests["99_openshift-lso_ns.yaml"] = localStorageNamespace
+	manifests["99_openshift-lso_operator_group.yaml"] = lsoOperatorGroup
+	manifests["99_openshift-lso_subscription.yaml"] = lsoSubscription
+	manifests["99_openshift-lso_lvset_cr.yaml"] = localVolumeSet
+	manifests["99_openshift-lso_lvset_crd.yaml"] = localVolumeSetCrd
+	return manifests
+}
+
+const lsoSubscription = `apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  channel: "4.6"
```

Comment:
What is the plan for managing this over time? Particularly in a future where multiple OCP releases are supported for installation?

Reply:
We can pass the OpenShift version to be installed from the Cluster and update the channel accordingly.
If the CRDs changes between versions, then the Manifests() method should handle the supported versions and return the right CRDs.

Reply:
Cluster have a OCP version so we can probably have this as template, but if there are changes don't include only parameters then we will need to have a map of configurations.